bot/converters.py

```diff
@@ -0,0 +1,39 @@
+from typing import Union
+
+from discord.ext import commands
+from discord.ext.commands import BadArgument, Context, Converter
+from discord.utils import escape_markdown
+
+SourceType = Union[commands.HelpCommand, commands.Command, commands.Cog, str, commands.ExtensionNotLoaded]
+
+
+class SourceConverter(Converter):
+    """Convert an argument into a help command, tag, command, or cog."""
+
+    @staticmethod
+    async def convert(ctx: Context, argument: str) -> SourceType:
+        """Convert argument into source object."""
+        if argument.lower() == "help":
+            return ctx.bot.help_command
```

Comment:
We don't have a subclassed help_command, what is the output of this?

Reply:
[discord.py does though](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Bot.help_command)

Reply:
Ah I see what you mean though, yes it may be problematic because it is defined inside the discord.py dependency.

Reply:
Yeah it wouldn't resolve the URL correctly with the current code, is what I was thinking